drivers/Makefile.dep

```diff
@@ -677,8 +677,8 @@ ifneq (,$(filter ws281x_%,$(USEMODULE)))
 endif
 
 ifneq (,$(filter ws281x,$(USEMODULE)))
-  FEATURES_OPTIONAL += arch_avr8
-  FEATURES_OPTIONAL += arch_native
+  FEATURES_REQUIRED_ANY += arch_avr8|arch_native
```

Comment:
Why does this get a special character (`|`) and other feature lists not? I think parsing (and declaring) will become way easier, if we just use whitespaces (the construct above need to take whitespaces into account anyway as when `FEATURES_REQUIRED_ANY` is already set to `something` it will become `something arch_avr8|arch_native`.

Reply:
I think that's intentional.
In that case it would require feature `something`  AND `arch_avr8` OR `arch_native`

Reply:
Lets say module A requires `arch_avr8` or `arch_native`. And module B requires `periph_i2c` or `periph_spi`. The condition would be `((arch_avr8 || arch_native) && (periph_i2c || periph_spi))`.

Having that separated by spaces, e.g.

```Makefile
FEATURES_REQUIRED_ANY += arch_avr8 arch_native
FEATURES_REQUIRED_ANY += periph_i2c periph_spi
```

Would result in `FEATURES_REQUIRED_ANY` having the content `arch_avr8 arch_native periph_i2c periph_spi`. So half of the information is lost.

Reply:
Ah, I see. Sorry for the noise then!

Reply:
No problem. Questioning whether odd/inconsistent design choices are really needed is certainly a good thing to do :-)